linux_os/guide/system/software/integrity/crypto/harden_ssh_client_crypto_policy/rule.yml

```diff
@@ -0,0 +1,36 @@
+documentation_complete: true
+
+title: 'Harden SSH client Crypto Policy'
+
+description: |-
+    Crypto Policies are means of enforcing certain cryptographic settings for selected applications including OpenSSH client.
+    The SSH client is by default configured to modify its configuration based on currently configured Crypto-Policy. However, in certain cases it might be needed to override the Crypto Policy specific to OpenSSH Client and leave rest of the Crypto Policy intact.
+    This is unfortunately not directly possible through the Crypto Policy mechanism due to the way in which the SSH Client parses configuration files. During the parsing process, as soon as the client parses some configuration option and its value, it remembers it and ignores any subsequent overrides. The mechanism of crypto policies appends eventual customizations at the end of the system wide crypto policy. Therefore, if the crypto policy customization overrides some parameter which is already configured in the system wide crypto policy, the SSH client will not honor that customized parameter.
+    Considering supported platforms, the default configuration file in <tt>/etc/ssh_ssh_config</tt> includes files matching the pattern <tt>/etc/ssh_ssh_config.d/*.conf</tt>. Currently, by default, there is present only one file called <tt>05-redhat.conf</tt>. This file subsequently includes the actual system-wide crypto policy placed in <tt>/etc/crypto-policies/back-ends/openssh.config</tt>.
+    To override the system wide crypto policy for Openssh client, place a file in the <tt>/etc/ssh/ssh_config.d/</tt> so that it is loaded before the <tt>05-redhat.conf</tt>. In this case it is file named <tt>02-ospp.conf</tt> containing parameters wchich need to be changed with respect to the crypto policy.
+    This rule checks if the file exists and if it contains required parameters and values which modify the Crypto Policy.
+
```

Comment:
I think the rule description is a bit convoluted.
By swapping paragraph in line 10 with paragraph in line 8, description of what to do to override the crypto policy is more evident.
After that, merge paragraph currently in lines 8 and 9 and  make them shorter. This way the justification of why it needs to be done this way is shorter and secondary.